seatunnel-connectors-v2/connector-hive/pom.xml

```diff
@@ -31,9 +31,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.apache.seatunnel</groupId>
-            <artifactId>seatunnel-hive-shade</artifactId>
-            <version>${project.version}</version>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-exec</artifactId>
+            <scope>provided</scope>
         </dependency>
```

Comment:
If we do, I think it's better to provide the  version requires, as well as reflect in the documentation that users need to provide it themselves.

Reply:
Yes, we need to specify the premise of running `hudi source` connector, User need to ensure that the spark/flink cluster has integrated hive. Currently, the tested hive version is 2.3.9